src/platform-integration/web/faq.md

```diff
@@ -100,17 +100,65 @@ although no such support is built in.
 
 ### How do I embed a Flutter web app in a web page?
 
-You can embed a Flutter web app,
-as you would embed other content,
-in an [`iframe`][] tag of an HTML file.
-In the following example, replace "URL"
-with the location of your hosted HTML page:
+You have two embed options: embed in an `iframe` and embed in a `div` container.
+
+#### Use an `iframe`
+
+To embed a Flutter web app in an [`iframe`][] tag of your HTML file,
+add the tag and link to the HTML file that contains your Flutter app.
+
+In the following example, replace `URL` with the location of your HTML page.
 
 ```html
 <iframe src="URL"></iframe>
 ```
 
-If you encounter problems, please [file an issue][].
+#### Use an `div` container
```

Comment:
I wonder if the term "div" can be taken more literally than it should. After all, `<div>` is a standard web element. The reason it shouldn't be taken literally is because we actually do not require that the element be a `<div>`.  There may be some requirements w.r.t. the elements attachedness, positioning, etc (@ditman can help us define those), but you should be able to use any random `<my-super-duper-element>` just fine.